bool ledValue
uint16 currentPotValue
